src/node_http_parser.cc

```diff
@@ -617,7 +617,8 @@ class Parser : public AsyncWrap, public StreamListener {
       enum http_errno err = HTTP_PARSER_ERRNO(&parser_);
 
       Local<Value> e = Exception::Error(env()->parse_error_string());
-      Local<Object> obj = e->ToObject(env()->isolate());
+      Local<Object> obj = e->ToObject(env()->isolate()->GetCurrentContext())
```

Comment:
ditto

Reply:
@addaleax what about this one?

Reply:
@targos I think it currently doesn’t matter because we only provide built-in native modules for the main context of an `Environment`… This would probably the correct choice if we were to change that. On the other hand, using `env()->context()` might be a tiny bit faster.

> This scope has an `EscapableHandleScope`, so maybe it'll be self documenting to use `scope.GetIsolate()`?

That would be very unidiomatic and.